kyo-core/shared/src/main/scala/kyo/Console.scala

```diff
@@ -19,28 +19,28 @@ final case class Console(unsafe: Console.Unsafe):
       * @param s
       *   The string to print.
       */
-    def print(s: String)(using Frame): Unit < (IO & Abort[IOException]) = IO.Unsafe(Abort.get(unsafe.print(s)))
+    def print(s: Text)(using Frame): Unit < (IO & Abort[IOException]) = IO.Unsafe(Abort.get(unsafe.print(s.show)))
```

Comment:
Should these methods accept `print[A](a: A)(using Frame, Render[A])`? Or is that the value of the `Conversion`?

Reply:
It does not do any conversion. You can pass a string, since `Text` is a supertype of `String`, or you can convert to text using `Render` or t-string interpolation.

@fwbrasil  how would feel about having these methods doing conversions based on `Render`? We could do what @hearnadam suggests and `use` a `Render[A]`, or we can accepted a `Rendered` value which will be converted automatically. I prefer @hearnadam 's suggestion, since it's more transparent, but I'm on the fence about changing it at all.

Reply:
we already have the overloaded `Console` methods that take objects and use `pprint` to generate the string. The formatted output with coloring is quite helpful to inspect more complex objects. Perhaps we could migrate everything to `Render` and follow up later improving its output like in `pprint`? It'd be nice to avoid the dependency as well